src/appkit.cs

```diff
@@ -1040,6 +1053,30 @@ interface NSApplicationDelegate {
 		[Deprecated (PlatformName.MacOSX, 11, 0, message: "Now optional on NSApplicationDelegate.")]
 		[Export ("application:delegateHandlesKey:"), DelegateName ("NSApplicationHandlesKey"), NoDefaultValue]
 		bool HandlesKey (NSApplication sender, string key);
+
+		[Mac (12,0)]
+		[IgnoredInDelegate]
+		[Export ("applicationSupportsSecureRestorableState:")]
+		bool ApplicationSupportsSecureRestorableState (NSApplication application);
+
+		[Mac (12,0)]
+		[IgnoredInDelegate]
+		[Export ("application:handlerForIntent:")]
+		[return: NullAllowed]
+		NSObject GetApplication (NSApplication application, INIntent intent);
```

Comment:
This is somewhat of a similar case, the important part is what comes after 'application:', so this is a better managed name:

```suggestion
		NSObject GetHandler (NSApplication application, INIntent intent);
```